frontend/admin/src/js/components/user/UserProfilePrintButton.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button } from "@common/components";
+import { commonMessages } from "@common/messages";
+import React, { useRef } from "react";
+import { useIntl } from "react-intl";
+import { useReactToPrint } from "react-to-print";
+import { Scalars, useGetUserProfileQuery } from "../../api/generated";
+import UserProfileDocument from "./UserProfileDocument";
+
+export interface UserProfilePrintButtonProps {
+  userId: Scalars["ID"];
+}
+
+export const UserProfilePrintButton: React.FunctionComponent<{
+  userId: Scalars["ID"];
+}> = ({ userId, children }) => {
+  const intl = useIntl();
+
+  // would be nice to only fire this if the button was clicked but you would need an imperative version of the query that you could await on
+  const [{ data: initialData, fetching, error }] = useGetUserProfileQuery({
+    variables: { id: userId },
+  });
+
+  const componentRef = useRef(null);
+  const handlePrint = useReactToPrint({
+    content: () => componentRef.current,
+    copyStyles: false,
+    pageStyle: `@page {
```

Comment:
I'd like to centralize some standard print styling in a file but I'm not sure how to get CSS into a string.  I tried CSS modules but that provides an object.  I starting looking into some webpack plugins to inline a file but decided it was more trouble than was worth.